### PR TITLE
fix: portfolio amount value

### LINKED ISSUE #1503: when we don't have network connection, don't display <$0.1
when we don't have network for now we have

IMG_0875

we should have instead

Screenshot 2024-06-19 at 23 18 16

### PR DESCRIPTION
**Pull request by Jaxiii**
## Changes

Replaces '<$0.01' to '-' when value is unknown

## Related issues

Fixes #1503

<details>
<summary>Screenshots & Videos</summary>
<br>

Simulator Screenshot - iPhone 15 Pro Max - 2024-06-23 at 14 51 37

Simulator Screenshot - iPhone 15 Pro Max - 2024-06-24 at 14 18 51

https://github.com/espresso-cash/espresso-cash-public/assets/17258221/39318708-1be5-4fde-b23f-a4ecd3f775bf

</details>

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [x] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

**Comment by romeo4934** [comment]
@Jaxiii Please remove the total value to $0.00 of crypto portofolio if token price are not available

File: packages/espressocash_app/lib/features/currency/models/amount.dart [hunk 1]
```diff
@@ -51,6 +54,8 @@ sealed class Amount with _$Amount {
 
   bool get isZero => decimal == Decimal.zero;
 
+  bool get isUnknown => value < 0;
```
Comment (ookami-kb) [inline packages/espressocash_app/lib/features/currency/models/amount.dart:57]: No, negative amount can be totally valid value, it shouldn't have a special meaning. What's wrong with having `null` as "unknown amount"?

Comment (ookami-kb) [reply packages/espressocash_app/lib/features/currency/models/amount.dart:54]: This should be removed.

Comment (Jaxiii) [reply packages/espressocash_app/lib/features/currency/models/amount.dart]: > This should be removed.

sorry, I must have forgotten to add the file to the commit.

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**